geom/alg_set_op.go

```diff
@@ -13,39 +13,38 @@ func Union(a, b Geometry) (Geometry, error) {
 	if b.IsEmpty() {
 		return a, nil
 	}
-	g, err := setOp(a, b, selectUnion)
+	g, err := setOp(a, b, selectUnion, false)
```

Comment:
Do you know the reason that union works without first merging members of geometry collections?